flake.nix

```diff
@@ -15,10 +15,11 @@
     ...
   }:
     flake-parts.lib.mkFlake {inherit inputs;} {
+      # Listed platforms must be available to the current machine for evaluating the Web UI packages
       systems = [
-        "aarch64-darwin"
-        "aarch64-linux"
-        "x86_64-darwin"
+        # "aarch64-darwin"
+        # "aarch64-linux"
+        # "x86_64-darwin"
```

Comment:
What is the reason for this?

Reply:
As the comment a couple of lines above explains, the Web UI packages contain the 'yarn2nix' package, which requires the listed systems to be available on the local machine during evaluation for some reason. Even if we had machines for remote building these kinds of packages, we can't assume that everyone has access to each platform.

Do you have some kind of solution to this? This is the current output with all the platforms listed:
```shell
[I] kari@torque ~/W/HomestakerOS (feat/pkgs-webui)> nix flake check
warning: Git tree '/home/kari/Workspace/HomestakerOS' is dirty
warning: unknown flake output 'schema'
Failed to find a machine for remote build!
derivation: fw0007wxjd7yqjjahv7zj4ng5z7z5hkx-yarn2nix-1.0.0.drv
required (system, features): (aarch64-darwin, [])
2 available machines:
(systems, maxjobs, supportedFeatures, mandatoryFeatures)
([aarch64-linux, i686-linux, x86_64-linux], 20, [benchmark, big-parallel, kvm, nixos-test], [])
([aarch64-linux, armv7l-linux], 16, [benchmark, big-parallel, gccarch-armv8-a, kvm, nixos-test], [])
error:
       … while checking flake output 'packages'

         at «none»:0: (source not available)

       … while checking the derivation 'packages.aarch64-darwin.default'

         at «none»:0: (source not available)

       (stack trace truncated; use '--show-trace' to show the full trace)

       error: a 'aarch64-darwin' with features {} is required to build '/nix/store/fw0007wxjd7yqjjahv7zj4ng5z7z5hkx-yarn2nix-1.0.0.drv', but I am a 'x86_64-linux' with features {benchmark, big-parallel, kvm, nixos-test}
```

Reply:
> I think it's fine to merge first

I will merge this after I solve the problem with the after build generated JSON data not being picked up by the frontend. Thats a pretty huge problem too.

> Does the output change if it's the systems is flakeExposed?

I tried this and if i recall correctly, it didnt change a thing.